data-ingestion-and-preparation/grafana-grafwiz.ipynb

```diff
@@ -211,33 +415,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Create a table and log viewer in one row\n",
+    "# Create a table and log viewer for the NoSQL table in one row\n",
     "tbl = Table('Current Stocks Value', span=12).source(table=stocks_kv_table,fields=['symbol','volume', 'price', 'last_updated'],container=v3io_container)\n",
     "dash.row([tbl])\n",
     "\n",
-    "# Metrics row will\n",
+    "# Create a TSDB metrics graph\n",
     "metrics_row = [Graph(metric).series(table=stocks_tsdb_table, fields=[metric], container=v3io_container) for metric in ['price','volume']]\n",
-    "\n",
-    "dashboard = dash.row(metrics_row)\n",
-    "\n"
+    "dashboard = dash.row(metrics_row)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Deploy to Grafana\n",
-    "dash.deploy(grafana_url)"
+    "<a id=\"grafwiz-grafana-dashboard-deploy\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploying the Dashboard to Grafana\n",
+    "\n",
+    "Deploy the new Grafana dashboard to your Grafana service.\n",
+    "\n",
+    "> **Note:** Allow a few minutes for the dashboard loading to complete. "
```

Comment:
Did it take you a few minutes?

Reply:
I didn't test it. I didn't add this text, I just edited and moved it from a previous cell, and I asked Arthur and Or to confirm the move on Slack (I'm still awaiting their reply; they're also reviewers in this PR). The is from the original NB:
```markdown
 #### Create a table and 2 graphs
 Please note that you have to wait for couple of minutes to see the graphs being updated
```